### PR TITLE
luci-app-lorawan-basicstation: Add luci-app-lorawan-basicstation

### PR DESCRIPTION
**Pull request by mars642**
LuCi support for Basicstation

Depends on https://github.com/openwrt/packages/pull/19188

Signed-off-by: Marcus Schref <mschref@web.de>

### CODE REVIEW

**Comment by mars642** [comment]
ping @aparcar

**Comment by aparcar** [comment]
Please post some screenshots

**Comment by mars642** [comment]
Here are some screenshots of the luci app
GeneralSettings1
GeneralSettings2
AdvancedSettings
LogMessages

File: applications/luci-app-basicstation/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the Apache License Version 2.0.
+# See https://www.apache.org/licenses/LICENSE-2.0 for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-basicstation
+PKG_VERSION:=2.0.6
```
Comment (BKPepe) [inline applications/luci-app-basicstation/Makefile:11]: Why such versioning? Couldn't it be 1.0.0 as this is the first initial version, which could gets into this repository?

**Comment by hnyman** [comment]
I would rename this to something like luci-app-lorawan-basicstation
And also add LoRaWAN to the title seen in menuconfig:
`LUCI_TITLE:=LuCI Support for basicstation`
--> `LUCI_TITLE:=LuCI Support for LoRaWAN basicstation`

Otherwise the "basic station" may sound too much like a simplified setup tool for a router.  (there is already a PR for luci-mod-simple, etc.)

File: applications/luci-app-basicstation/Makefile (hunk 1 above)
Comment (hnyman) [reply applications/luci-app-basicstation/Makefile]: Removing versioning is fine, as there is automatic versioning in LuCI. (most packages do not have manual versions at all)

**Pull request merged**